jobs/suade_senior_developer.html

```diff
@@ -0,0 +1,62 @@
+---
+title: Senior Developer
+company: Suade Labs
+location: London, England
+url: https://suade.org/jobs
+contract: permanent
+contact:
+name: Jesper Thomschutz
```

Comment:
This looks wrong - name and email should be indented. You need to follow the template exactly!